### PR TITLE
fix: be more safe checking for presence of styles

### PR DESCRIPTION
**Pull request by pauldambra**
We were checking 

`if (wireframe.y) {`

so of course, 0 values never made it into the block.

Let's be more explicit about what we are checking/expecting

### CODE REVIEW

File: ee/frontend/mobile-replay/wireframeStyle.ts [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+
+function isString(candidate: unknown): candidate is string {
+    return typeof candidate === 'string'
+}
+
+function isUnitLike(candidate: unknown): candidate is string | number {
+    return isNumber(candidate) || (isString(candidate) && candidate.length > 0)
+}
+
 function ensureUnit(value: string | number): string {
```
Comment (marandaneto) [inline ee/frontend/mobile-replay/wireframeStyle.ts:15]: This method can reuse `isNumber` now

File: ee/frontend/mobile-replay/wireframeStyle.ts [hunk 2]
```diff
@@ -1,17 +1,33 @@
 import { MobileStyles, wireframe } from './mobile.types'
 
+function isNumber(candidate: unknown): candidate is number {
```
Comment (marandaneto) [inline ee/frontend/mobile-replay/wireframeStyle.ts:3]: Probably not in this PR but the codebase has many of those ` === 'number'` checks, maybe making this global/reusable and refactoring would make sense.

Comment (marandaneto) [reply ee/frontend/mobile-replay/wireframeStyle.ts:3]: Same for `=== 'string'` :D

**Review by marandaneto — APPROVED** [review]

File: ee/frontend/mobile-replay/wireframeStyle.ts (hunk 2 above)
Comment (pauldambra) [reply ee/frontend/mobile-replay/wireframeStyle.ts:3]: Yeah, was thinking this as I pulled them to one side.

We don't care so much in this codebase, but type checking into one place in posthog-js reduced bundle by about 1%.

**Pull request merged**